.github/workflows/release.yml

```
@@ -0,0 +1,424 @@
+          echo "Verifying Linux AMD64 binary..."
+          ./goingenv-$GOOS-$GOARCH --version
+        else
+          echo "✅ Cross-compiled binary for $GOOS/$GOARCH (verification skipped - incompatible platform)"
+        fi
+        
+        echo "✅ Binary built successfully"
+
+    - name: Create archive
+      run: |
+        ARCHIVE_NAME="goingenv-v${{ needs.validate.outputs.version }}-${{ matrix.goos }}-${{ matrix.goarch }}.tar.gz"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)